extensions/wikia/Email/templates/greeting.mustache

```diff
@@ -0,0 +1,4 @@
+<div class="greeting">
+	<p>{{salutation}}</p>
+	<p>{{{summary}}}</p>
```

Comment:
Nothing wrong here, but just a heads up that there'll be a conflict with SOC-459 where Garth added ```strong``` tags around the summary.